devicetypes/smartthings/aeon-multisensor-gen5.src/aeon-multisensor-gen5.groovy

```diff
@@ -181,7 +181,7 @@ def zwaveEvent(physicalgraph.zwave.commands.sensormultilevelv5.SensorMultilevelR
 			break;
 		case 3:
 			map.name = "illuminance"
-			map.value = cmd.scaledSensorValue.toInteger()
+			map.value = isZooz() ? getLuxFromPercentage(cmd.scaledSensorValue.toInteger()) : cmd.scaledSensorValue.toInteger()
```

Comment:
This capability is handled locally for this dth so this code path won't be hit

Reply:
@tpmanley It should get bounced back up to cloud correctly if they've set their scale bit correctly. The LE code checks for the scale bit.

@PKacprowiczS can you expand this case to add handling based on `cmd.scale` (instead of just device)? `0x0` is percentage and `0x1` is lux. I understand devices will probably have different scales in the future (and probably won't be put on this DTH) but it looks more standardized and we can confirm that they're sending correct z-wave packets.

Reply:
Illuminance is a smart lighting trigger so it won't work if it executes in the DTH

Reply:
the trigger won't fire at all, even if the hub is online?

Reply:
Right, if the behavior is believed to run locally it will not run in the cloud even if the DTH generates the trigger event

Reply:
😞 so what should we do? This mapping of %=>lux doesn't seem robust enough to include in hubcore and breaking this whole device into a new DTH because of this seems like a waste, but we can't leave this as-is since illumination triggers just won't work.

So a new DTH seems like our only option...

Reply:
Unless @varzac has any other ideas I think a new DTH is the best option

Reply:
Yeah, unfortunately I'd say new DTH is the only option.  I can't think of a better solution.